### ISSUE TITLE
[Calendar | a11y]: Week day and week numbers are not announced by screen readers

### ISSUE THREAD

**Issue by PlutaKatarzyna** [SAP/ui5-webcomponents#10002]
### Bug Description

While navigating through calendar with keyboard, information about week number and week day are not read by screen readers.

UDEx bug ticket reported by accessibility testers team:
https://github.tools.sap/sapudex/digital-design-system/issues/926

### Affected Component

ui5-calendar

### Expected Behaviour

Add Aria-label or aria-describedby to a single calendar day, to contain information about weekday and week.

### Steps to Reproduce

1. Enable any screen readers (Jaws for Windows or VoiceOver for macOS)
2. Go to UI5 Date Picker component and open calendar
3. Use keyboard to change selected day
4. Read/hear screenreader information

### Priority

Medium

### UI5 Web Components Version

1.24.8

### Browser

Chrome

### Organization

SAP / UDEx components team

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by unazko** [comment]
Hello @PlutaKatarzyna,

This issue is the same as this previously reported issue:
https://github.com/SAP/ui5-webcomponents/issues/9441

The same isn't reproducible with NVDA screen reader. It is a JWAS issue logged under number 0093 in wiki.
Unfortunately there is nothing we can do at our side as it is a third party issue.

Best regards,
Boyan

**Comment by unazko** [comment]
Hi @PlutaKatarzyna,

I've performed some testing with JAWS and NVDA.
The week and day information isn't announced by JAWS with both VPC on and off when entering in the grid.
Also that would happen even if the `grid` isn't nested into a `dialog`:
https://jsfiddle.net/unazko/h06j1td8/2/
The week and day announcements are present with JAWS when navigating with the arrow keys in the grid.
I've already provided the new data from the testing, in order to update the JAWS issue description.

I can confirm that the issue isn't reproducible with NVDA.
You could also watch and listen to the provided video representing the testing results with NVDA:
https://github.tools.sap/sapudex/digital-design-system/issues/926#issuecomment-8584491
and all relevant information is properly conveyed.

That is why I've got confused from the statement that issue is reproducible with NVDA.
Still this is a JAWS issue as we do expect the day name and week number to be announced as a "rowheader" and a "columnheader" in a "row" and part of a "grid" element.

Best regards,
Boyan
